packages/eslint-plugin/src/rules/use-pipe-transform-interface.ts

```diff
@@ -1,45 +1,106 @@
 import { TSESTree } from '@typescript-eslint/experimental-utils';
+import { RuleFixer } from '@typescript-eslint/experimental-utils/dist/ts-eslint';
```

Comment:
@JamesHenry Could we have `RuleFixer` exported directly from the main entry point? This deep import just doesn't feels good for me.

Reply:
Yeah, very true, please could you open an issue on typescript-eslint, and add a TODO comment referencing it so we don't forget in future?

Reply:
I created the issue https://github.com/typescript-eslint/typescript-eslint/issues/2931 and put a comment here, take a look please.

Reply:
I've seen Brad's response, shows how rusty I am :) Please can we update this accordingly